### DISCUSSION TITLE
Give option to re-calculate features for a fragment

### DISCUSSION THREAD

**Discussion by jkosinski** [KosinskiLab/AlphaPulldown#606] [Ideas]
This is really for later and to discuss first:

Sometimes we do want to re-calculate MSA and templates for a chopped fragment, so create_features should have an option to take prot_A,start-end as input and create features that preserve numbering.

**Comment by DimaMolod** [comment]
The current chopped-object path already preserves numbering when slicing existing full-length features, via residue_index slicing in objects.py. What does not exist is a region-aware recalculation path in feature generation: create_individual_features.py only iterates raw FASTA entries. So this is not “add fragment support to ChoppedObject”, it is “invent a new fragment-aware feature-generation mode and then patch numbering back onto newly computed features.”
Hard to implement, dubious added value. Moving to 'Ideas'
